modules/dataflow_bucket/variables.tf

```
@@ -0,0 +1,12 @@
+variable "project_id" {
+  description = "The project_id to deploy the example instance into.  (e.g. \"simple-sample-project-1234\")"
+}
+
+variable "region" {
+  description = "The GCS bucket region. This should be the same as your dataflow job's zone ot optimize performance."
+  default     = "us-central1"
+}
+
+variable "bucket_name" {
```

This name is redundant given the module name.

```suggestion
variable "name" {
```

---

Fixed in https://github.com/terraform-google-modules/terraform-google-dataflow/pull/1/commits/b42d51328a83653787d5973ab4ea414abd0a5bb1